src/GovUk.Education.ExploreEducationStatistics.Content.Model/ReleaseStatus.cs

```diff
@@ -0,0 +1,25 @@
+#nullable enable
+
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class ReleaseStatus
+    {
+        public Guid Id { get; set; }
+
+        public Guid ReleaseId { get; set; }
+
+        public Release Release { get; set; }
+
+        public string InternalReleaseNote { get; set; }
+
+        public ReleaseApprovalStatus ApprovalStatus { get; set; }
+
+        public DateTime? Created { get; set; }
```

Comment:
I think you are missing the UTC date/time conversion on this field in the ContentDbConfig. See how other datetime fields are configured.